data_model/src/events/data/events.rs

```diff
@@ -351,6 +351,8 @@ mod account {
         Deleted(AccountId),
         AuthenticationAdded(AccountId),
         AuthenticationRemoved(AccountId),
+        AssetRegisterTxAdded(AccountId),
+        AssetRegisterTxRemoved(AccountId),
```

Comment:
I strongly believe, that it should happen on every `Register<AssetDefinition>`, so I think it's obvious and don't deserve any separate events